torch/distributed/distributed_c10d.py

```diff
@@ -767,6 +785,32 @@ def _new_process_group_helper(
                     )
             _pg_map[pg] = (Backend.NCCL, store)
             _pg_names[pg] = group_name
+        elif backend == Backend.UCC:
+            if not is_ucc_available():
+                raise RuntimeError("Distributed package doesn't have UCC built in")
+
+            pg = ProcessGroupUCC(prefix_store, rank, world_size, timeout=timeout)
+            # In debug mode and if GLOO is available, wrap in a wrapper PG that
+            # enables enhanced collective checking for debugability.
+            if get_debug_level() == DebugLevel.DETAIL:
+                if not _GLOO_AVAILABLE:
+                    logger.info(
+                        """TORCH_DISTRIBUTED_DEBUG was set to DETAIL, but
+                                GLOO is not available. Build with Gloo to
+                                create a wrapper process group in debug mode
+                                to aid collective desynchronization debugging."""
+                    )
```

Comment:
```suggestion
            # In debug mode and if GLOO is available, wrap in a wrapper PG that
            # enables enhanced collective checking for debugability.
            if get_debug_level() == DebugLevel.DETAIL:
                if not _UCC_AVAILABLE:
                    logger.info(
                        """TORCH_DISTRIBUTED_DEBUG was set to DETAIL, but
                                UCC is not available. Build with UCC to
                                create a wrapper process group in debug mode
                                to aid collective desynchronization debugging."""
                    )
```

Reply:
I think Gloo is the correct one as that's a separate PG being used as a wrapper PG for debugging (see the same logic with NCCL PG right above these lines).